internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -296,6 +297,33 @@ func (e *BootstrapEnvStackConfig) ToEnv(stack *cloudformation.Stack) (*config.En
 	}, nil
 }
 
+func (e *EnvStackConfig) cdnConfig() template.CDNConfig {
```

Comment:
nit we could return `*template.CDNConfig` to simplify the logic in this function and use

```
{{- if .CDNConfig}}
```

in the template file.